static/app/components/links/listLink.tsx

```diff
@@ -50,9 +56,11 @@ function ListLink({
       className={classNames({[activeClassName]: active}, className)}
       disabled={disabled}
     >
-      <RouterLink {...props} onlyActiveOnIndex={index} to={disabled ? '' : target}>
-        {children}
-      </RouterLink>
+      <Tooltip title={disabled ? disabledTooltip : null}>
```

Comment:
Can we move this to `focusTabs` and wrap the label with Tooltip? Otherwise ListLink can become bloated with the number of props (imagine if we needed to customize the props on Tooltip).

Reply:
done!